spec/dummy/config/database.sample.yml

```diff
@@ -10,14 +10,14 @@ base: &base
 
 development:
```

Comment:
@Fryguy when using `bin/console` it uses `config/database.yml` so this change makes running tests easier, but confuses the same for `bin/console`  Is there any fix that we could use for both? Or possibly a better error message letting the user know what isn't configured correctly

Reply:
I did `bundle exec rails console` and it worked ... can you try that?

Reply:
That works.  I didn't know `rails c` was available.  I've been using `bin/console`. Probably means `bin/console` can be removed.